manifests/golang.yml

```diff
@@ -284,6 +284,7 @@ tests/:
       Test_Debugger_Line_Probe_Statuses: missing_feature
       Test_Debugger_Method_Probe_Snaphots: irrelevant
       Test_Debugger_Method_Probe_Statuses: missing_feature
+      Test_Debugger_Mix_Log_Probe: irrelevant
```

Comment:
It means it won't never be implemented in golang ?

Reply:
It will be implemented eventually, but not in the plan for a really long time.